css/index.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  margin: 0 auto;
+  padding: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+  counter-reset: day 0;
+}
+.calendar .calendar__day {
+  width: 100px;
+  height: 100px;
+  border: solid 1px black;
+  background-color: #eee;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  position: relative;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+  cursor: pointer;
+  -webkit-transition: linear 0.5s;
+  transition: linear 0.5s;
```

Comment:
Consider removing the `-webkit-transition` prefix unless you need to support very old versions of WebKit browsers. Modern browsers support the unprefixed `transition` property.